OutOfSchool/OutOfSchool.DataAccess/Repository/ChangesLogRepository.cs

```diff
@@ -49,8 +49,8 @@ public ChangesLogRepository(OutOfSchoolDbContext dbContext)
             if (propertyName == "InstitutionId")
             {
                 propertyNameTmp = "Institution";
-                oldValueTmp = dbContext.Institutions.Where(x => x.Id == Guid.Parse(oldValue)).SingleOrDefault().Title;
-                newValueTmp = dbContext.Institutions.Where(x => x.Id == Guid.Parse(newValue)).SingleOrDefault().Title;
+                oldValueTmp = dbContext.Institutions.Where(x => x.Id == Guid.Parse(oldValue)).SingleOrDefault()?.Title;
+                newValueTmp = dbContext.Institutions.Where(x => x.Id == Guid.Parse(newValue)).SingleOrDefault()?.Title;
```

Comment:
(Code style) You could simplify this code moving predicate from `.Where()` to `.SingleOrDefault()`